static/main.html

```diff
@@ -14,31 +15,48 @@
     <h1>Wicked to NetworkManager</h1>
   </header>
   <div id="content">
-    <div id="input-container">
-      <div id="files-header">
-        <h3>Wicked configuration files</h3>
-        <div id="files-header-buttons">
-          <label for="file-upload" class="custom-button">Upload files</label>
-          <input id="file-upload" type="file" multiple />
-          <label for="reset-files-button" class="custom-button">Reset files</label>
-          <button id="reset-files-button">Reset files</button>
-        </div>
-      </div>
-      <div id="drop-area">
-        <div class="center-container" id="file-placeholder">
-          <div class="center-element">
-            <i class="placeholder fa-solid fa-file" id="file-placeholder-icon"></i>
-            <p class="placeholder">Drop ifcfg-* or .xml files here</p>
+    <div id="migration-files">
+      <div id="input-container">
+        <div id="files-header">
+          <h3>Wicked configuration files</h3>
+          <div id="files-header-buttons">
+            <label for="file-upload" class="custom-button">Upload files</label>
+            <input id="file-upload" type="file" multiple />
+            <label for="reset-files-button" class="custom-button">Reset files</label>
+            <button id="reset-files-button">Reset files</button>
           </div>
         </div>
-        <div id="file-container">
+        <div id="drop-area">
+          <div class="center-container" id="file-placeholder">
+            <div class="center-element">
+              <i class="placeholder fa-solid fa-file" id="file-placeholder-icon"></i>
+              <p class="placeholder">Drop ifcfg-* or .xml files here</p>
+            </div>
+          </div>
+          <div id="file-container">
+          </div>
         </div>
       </div>
-      <div id="migration-container">
-        <pre id="user-info"></pre>
-        <div>
-          <button type="submit" id="submit-button">Migrate</button>
+
+      <div id="migration-result-container">
+        <div id="files-header">
+          <h3>Configured files</h3>
```

Comment:
To better mirror the wicked files header
```suggestion
          <h3>NetworkManager configuration files</h3>
```